File: PRAKTIKUM-PEMROGRAMAN1/MODUL1/Python/PRAK110-2410817210024-MuhammadIbnuSina.py
import math
alas=5
tinggi=12
miring=math.sqrt(alas**2+tinggi**2)
keliling=tinggi+miring+alas
luas=0.5*alas*tinggi

print("Diketahui :")
print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm")
print("\nJawab :")
print(f"Sisi A = {tinggi} cm")
print(f"Sisi B = {miring:.0f} cm")
print(f"Sisi C = {alas} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm")

File: PRAKTIKUM-PEMROGRAMAN1/MODUL1/C/PRAK110-2410817210024-MuhammadIbnuSina.c
#include <stdio.h>
#include <math.h>

int main(){
    int alas=5, tinggi=12, miring, keliling, luas;
    miring=sqrt((alas*alas)+(tinggi*tinggi));
    keliling=tinggi+miring+alas;
    luas=0.5*alas*tinggi;

    printf("Diketahui : \n");
    printf("Alas = %d cm\n",alas);
    printf("Tinggi = %d\n",tinggi);
    printf("\nJawab : \n");
    printf("Sisi A = %d cm\n",tinggi);
    printf("Sisi B = %d cm\n",miring);
    printf("Sisi C = %d cm\n",alas);
    printf("Keliling = %d cm\n",keliling);
    printf("Luas = %d cm",luas);
    return 0;
}